### PR TITLE
Subclass searches

### PR DESCRIPTION
**Pull request by jamshark70**
These 2 commits fix ugly error messages that could be posted for invalid class browser searches that depend on a root class to be selected.

Usability improvement -- changes no core functionality.

### CODE REVIEW

File: SCClassLibrary/Common/GUI/PlusGUI/Core/ClassBrowser.sc [hunk 1]
```diff
@@ -263,7 +263,27 @@ ClassBrowser {
...
+						.action_({ w.close })
+						.maxWidth_(80),
+						StaticText()
+					)
+				))
+				.endFrontAction_({
+					w.endFrontAction_(nil).close;
+				})
+				.front;
+				^this  // early exit
```
Comment (crucialfelix) [inline SCClassLibrary/Common/GUI/PlusGUI/Core/ClassBrowser.sc]: I don't trust using ^ in a function.   it doesn't read clearly and I honestly fear that there are language bugs. 

```^warning.value``` would be better style

**Comment by jamshark70** [comment]
@crucialfelix -- Good point (though I use Function:block all over the place in my own code, and I never had a problem -- but agreed, stricter standard for the main class library). I've updated the code. Better?

**Comment by jamshark70** [comment]
@crucialfelix -- Good point (though I use Function:block all over the place in my own code, and I never had a problem -- but agreed, stricter standard for the main class library). I've updated the code. Better?

**Comment by crucialfelix** [comment]
thanks

**Pull request merged**